experiments/ClimaEarth/user_io/postprocessing.jl

```diff
@@ -29,11 +28,15 @@ function postprocess_sim(cs, postprocessing_vars)
     # Plot all model states and coupler fields (useful for debugging)
     !CA.is_distributed(cs.comms_ctx) && debug(cs, artifact_dir)
 
-    # If we have monthly data, plot the leaderboard
-    if t_end > 84600 * 31 * 3 && output_default_diagnostics
```

Comment:
`output_default_diagnostics` is now unused in ClimaCoupler. We might get an error from missing variables here if `output_default_diagnostics == false` and we try to plot the leaderboard, but I'm not entirely sure. It's a ClimaAtmos argument, so either way we can still include it in our config files, but if we don't use it here we can also remove it from `get_atmos_args`.

Reply:
Yes, you could get an error if `output_default_diagnostics == false` and don't have all the variables. But you could also have enough variables with `output_default_diagnostics == false`, meaning that this is not a good flag to check if we can plot the leaderboard or not. 

I removed it from `get_atmos_args`.